orders/models.py

```diff
@@ -0,0 +1,79 @@
+from django.db import models
+from django.utils import timezone
+from daycare.models import DayCare
+from dogowner.models import DogOwner
+
+
+class StatusOptions(models.TextChoices):
+    Pending = 'P', 'Pending'
+    Approved = 'A', 'Approved'
+    Canceled = 'C', 'Canceled'
+    Finished = 'F', 'Finished'
+    OnGoing = 'O', 'On Going'
+
+
+class Order(models.Model):
+    dog_owner_id = models.ForeignKey(DogOwner, on_delete=models.CASCADE)
+    daycare_id = models.ForeignKey(DayCare, on_delete=models.CASCADE)
+    booking_date = models.DateField(auto_now=True)
+    approving_date = models.DateField(blank=True, null=True)
```

Comment:
nit: `date_approved` or `approval_date` or `approved_on` are better naming conventions then using present progressive.

Reply:
Changed it to approval_date and cancellation_date.